docs/protocol/architecture.mdx

```diff
@@ -0,0 +1,149 @@
+---
+sidebar_position: 3
+title: "Architecture"
+---
+
+import Tooltip from '@site/src/components/Tooltip';
+
+# Architecture
+
+## System Overview
+
+The <Tooltip content="The on-chain smart contract protocol.">Nitrolite</Tooltip> protocol architecture consists of multiple layers working together to enable scalable, secure <Tooltip content="A secure communication pathway between participants that locks funds in an on-chain smart contract while enabling off-chain state updates.">state channel</Tooltip> operations:
+
+```mermaid
+graph TB
+    subgraph Application["Application Layer"]
+        APP[Chess, DEX, Gaming, Payments, Custom Logic]
+    end
+    
+    subgraph OffChain["Off-Chain Layer (Fast Updates)"]
+        CLIENT[Client SDK]
+        BROKER[Clearnode]
+        RPC[Nitro RPC WebSocket]
```

Comment:
We don't need to mention here websocket. I'm not sure whether we need to declare this entity here at all, as it's not referenced anywhere

Reply:
Maybe that's just a typo, make sure to check how it is rendered on the web page